moml/data/spice_dataset.py

```
@@ -0,0 +1,158 @@
+import torch
+import numpy as np
+import h5py
+import os
+from typing import List, Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)